### DISCUSSION TITLE
Accessing Windows via <hostname>.mshome.net?

### DISCUSSION THREAD

**Discussion by growlily** [microsoft/WSL#5861] [General]
Running Windows 10 19042, WSL 2. When accessing Windows networking apps, many guides tell us to grab IP from resolv.conf, which changes every time.

Found that I can access Windows via `<hostname>.mshome.net`. Seems a hosts.ics file in the same folder as hosts is automatically generated so this works. But I didn't find any document describing this behavior, wonder if this is a offcially supported approach?

**Comment by Tibalt** [comment]
it is different in win 10 pro wsl 2 with hyper V management function enabled.  

**hostname.local** points to win host's wsl adapter in wsl. 
**hostname.mshome.net** points to win host's default switch of hyper V adapter in wsl
